src/signals/IncidentMap/components/AddressLocation/AddressSearchMobile.tsx

```diff
@@ -51,6 +51,8 @@ export const AddressSearchMobile = ({
             variant="blank"
           />
           <StyledPDOKAutoSuggest
+            aria-label="searchAddressBarMob"
```

Comment:
@vdegraaf According to accessibility rules an input needs to have a label. Not having it gave an error when checking for accessibility. We didn't actually want to see the innerHTML so I hid it. It's not the most beautiful solution ever I concur

Reply:
@SireeKoolenWijkstra if the context of the input provides enough information to know what the input is used for there are several option to provide guidance for assistive technology. One is to hide the Label element indeed, but according to the docs an aria-label is also sufficient. Since the component has that already, the Label can be removed.

Reply:
My checker for accessibility gave an error for this, so I do think that we should have the label. Since the surrounding content is a StyledPDOKAutoSuggest and not a button or something like that, I presume the content is not clear and that's why we still got the error.